uvicorn/protocols/websockets/websockets_impl.py

```diff
@@ -201,6 +201,11 @@ async def run_asgi(self):
         """
         try:
             result = await self.app(self.scope, self.asgi_receive, self.asgi_send)
+        except (concurrent.futures.CancelledError, asyncio.CancelledError):
```

Comment:
I dont understand the `concurrent.futures.CancelledError` here, why isnt `asyncio.CancelledError` enough ?